---
layout: default
title: CV
permalink: /CV/
---

## Curriculum Vitae

[[PDF version]](/downloads/CV-MoHuang.pdf){:target="_blank"}


<h2 style="text-align: center;font-weight:bold">
Mo Huang, Ph.D.
</h2>

<h4 style="text-align: center;font-weight:normal">
Senior Scientist, Biostatistics<br>
Merck<br>
Email: mohuangx@gmail.com<br>
Website: mohuangx.github.io<br>
Last Updated: December 2021
</h4>

### **Education**

* **Ph.D. Statistics**, The Wharton School, University of Pennsylvania, 2020  
Advisor: Nancy R. Zhang

* **B.A. Mathematics**, Washington University in St. Louis, 2015

### **Experience**

* **Senior Scientist, Biostatistics**  
Merck  
*2020 -- Present*

* **PhD Candidate**  
Department of Statistics, The Wharton School, University of Pennsylvania  
*2015 -- 2020*

* **Biostatistics Summer Intern**  
Genentech  
*Summer 2018*

* **Undergraduate Researcher/Amgen Scholar**  
McDonnell Genome Institute at Washington University  
*2013 -- 2015*

* **Summer Intern**  
National Institutes of Health  
*Summers 2012 -- 2013*

### **Honors and Awards**

* Donald S. Murray Prize for Excellence in Teaching, 2019
* NSF Graduate Fellowship, 2016 -- 2019
* Judith & William Bollinger Fellowship, 2018
* Florence Moog Fellowship in Biological Sciences, 2011 -- 2015

### **Publications**

* Reeder J., **Huang, M.**, Kamingker, J.S., Paulson, J.N. (2021).
*MicrobiomeExplorer*: an R package for the analysis and visualization of microbial communities.
*Bioinformatics, 37* (9), 1317-1318.

* **Huang, M.** (2020).
A Statistical Framework for Denoising Single-cell RNA Sequencing Data (Dissertation). *PQDT-UK & Ireland*.

* **Huang, M.**, Zhang, Z., Zhang, N.R. (2020).
Dimension reduction and denoising of single-cell RNA sequencing data in the presence of observed confounding variables.
*bioRxiv*.

* Wang, J., Agarwal, D., **Huang, M.**, Hu, G., Zhou, Z., Ye, C., Zhang, N.R. (2019).
Data Denoising with transfer learning in single-cell transcriptomics. *Nature Methods, 16*(9), 875-878.

* **Huang, M.**, Wang, J., Torre, E., Dueck, H., Shaffer, S., Bonasio, R., … Zhang, N. R. (2018). 
SAVER: Gene expression recovery for single-cell RNA sequencing. *Nature Methods, 15*(7), 539–542.

* Wang, J., **Huang, M.**, Torre, E., Dueck, H., Shaffer, S., Murray, J., … Zhang, N. R. (2018). 
Gene Expression Distribution Deconvolution in Single Cell RNA Sequencing. 
*Proceedings of the National Academy of Sciences of the United States of America, 115*(28), E6437-E6446.

* Jayasinghe, R. G., Cao, S., Gao, Q., Wendl, M. C., Vo, N. S., Reynolds, S. M., … Ding, L. (2018). 
Systematic Analysis of Splice-Site-Creating Mutations in Cancer. *Cell Reports, 23*(1), 270–281.e3.

* Jin, T., **Huang, M.**, Jiang, J., Smith, P., & Xiao, T. S. (2018). 
Crystal structure of human NLRP12 PYD domain and implication in homotypic interaction. *PLOS ONE, 13*(1), e0190547.

* Wyczalkowski, M. A., Wylie, K. M., Cao, S., Mclellan, M. D., Flynn, J., **Huang, M.**, … Ding, L. (2017). 
BreakPoint Surveyor: A pipeline for structural variant visualization. *Bioinformatics, 33*(19), 3121–3122. 

* Jin, T., Chuenchor, W., Jiang, J., Cheng, J., Li, Y., Fang, K., … Xiao, T. S. (2017). 
Design of an expression system to enhance MBP-mediated crystallization. *Scientific Reports, 7*(January), 40991. 

* Jin, T., **Huang, M.**, Smith, P., Jiang, J., & Xiao, T. S. (2013). 
Structure of the caspase-recruitment domain from a zebrafish guanylate-binding protein. 
*Acta Crystallographica Section F Structural Biology and Crystallization Communications, 69*(8), 855–860. 

* Jin, T., **Huang, M.**, Smith, P., Jiang, J., & Xiao, T. S. (2013). 
The structure of the CARD8 caspase-recruitment domain suggests its association with the FIIND domain 
and procaspases through adjacent surfaces. 
*Acta Crystallographica Section F Structural Biology and Crystallization Communications, 69*(5), 482–487. 

### **Presentations**

* Joint Statistical Meetings, July 2019
* Joint Statistical Meetings, July 2018
* American Society of Human Genetics Meeting, October 2017

### **Programming Languages**

* R (expert)
* Python, Keras, PyTorch (proficient)
* Bash (proficient)
* SQL (competent)